pandas/core/common.py

```diff
@@ -2097,3 +2097,35 @@ def _random_state(state=None):
     else:
         raise ValueError("random_state must be an integer, a numpy "
                          "RandomState, or None")
+
+
```

Comment:
put in ``pandas.types.missing.py`` (new file); let's not add anymore to core/common.py